### ISSUE TITLE
Gesture detect is messed up using SideDrawer, Please fix them!

### ISSUE THREAD

**Issue by eeandrew** [telerik/nativescript-ui-samples-angular#12]
It seems The TAP event only works with Button!! If I add TAP to Label, like below, the TAP event will never be triggered on Label. Please take a look!!!!
```
<template drawerMain>
        <StackLayout>
            <Label text="OPEN DRAWER"  (tap)=openDrawer() textWrap="true" class="drawerContentText"></Label>
            <Button text="OPEN DRAWER" (tap)=openDrawer() ></Button>
        </StackLayout>
   </template>
```

**Comment by eeandrew** [comment]
Another problem, TOUCH is never triggered as well. Do you really test nativescript-ui? Please keep in mind that we actually pay to get this SDK, DO NOT screw it up!!

**Comment by VladimirAmiorkov** [comment]
HI @eeandrew ,

This was a know issue in the 1.2.x version which has been resolved in the latest **1.3.1** of [nativescript-telerik-ui](https://www.npmjs.com/package/nativescript-telerik-ui). Please give it a try.
